pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "azure-operator"
 	source      string = "https://github.com/giantswarm/azure-operator"
-	version            = "5.18.1-dev"
+	version            = "5.17.1-migratemps"
```

Comment:
Nope, `5.18.1-dev` is correct

Reply:
Changed to `5.18.1-migratemps`, I'll revert to `5.18.1-dev` when I finish testing.